modules/BTBidAdapter.js

```diff
@@ -65,6 +66,22 @@ function request(buildRequest, imps, bidderRequest, context) {
   return request;
 }
 
+/**
+ * Processes a bid response using the provided build function, bid, and context.
+ *
+ * @param {Function} buildBidResponse - The function to build the bid response.
+ * @param {Object} bid - The bid object to include in the bid response.
+ * @param {Object} context - The context object containing additional information.
+ * @returns {Object} - The processed bid response.
+ */
+function bidResponse(buildBidResponse, bid, context) {
+  const bidResponse = buildBidResponse(bid, context);
+  const { seat } = context.seatbid || {};
```

Comment:
does our server return 1 bid per request? asking since Kevel returns up to 3 bids and their `seatbid` is an array.
just out of curiosity, do you know why BE uses `seatbid` instead of `seatBid`

Reply:
it returns array of seatbids each object of which can contain several bids in its bid array.  You can see that in our tests https://github.com/blockthrough/Prebid.js/blob/feature/PE-120--Send-Prebid-Bidder-info-to-BT-Server/test/spec/modules/BTBidAdapte_spec.js#L106